lib/opensusebasetest.pm

```diff
@@ -639,6 +639,12 @@ sub post_fail_hook {
     # see any interesting console logs.
     send_key 'esc';
     save_screenshot;
+
+    # Find out in post-fail-hook if system is I/O-busy, poo#35877
+    select_console("root-console");
+    my $io_status = script_output("sed -n 's/^.*da / /p' /proc/diskstats | cut -d' ' -f10");
+    record_info('System I/O status:', 'is busy')     if ($io_status =~ qr/\w*1\w*/);
+    record_info('System I/O status:', 'is not busy') if ($io_status =~ qr/\w*0\w*/);
```

Comment:
I suggest to replace the two lines with

```
record_info('System I/O status:', ($io_status =~ /^0$/) ? 'idle' : 'busy')
```

because the value can be higher than 1 and to write it in a more simple way. However I would also output the actual value. Otherwise we can just do the check within the SUT and use `script_run` to report back the boolean result.

Reply:
updated and it works fine:
http://f40.suse.de/tests/48#step/logs_from_installation_system/25